release/bin/macos/rttr.command

```diff
@@ -3,16 +3,17 @@
 cd $(dirname $0)
 
 if [ ! "$1" = "interminal" ] ; then
+	scriptPath=$(pwd)
 	echo "#!/bin/sh" > /tmp/rttr.command
-	echo "$0 interminal" >> /tmp/rttr.command
+	echo "$scriptPath/$0 interminal" >> /tmp/rttr.command
```

Comment:
No. Only the invocation which can be relative. Manually calling it is impossible like that. So need basename $0